src/main/java/org/openrewrite/maven/RewriteDryRunMojo.java

```diff
@@ -96,6 +102,10 @@ public void execute() throws MojoExecutionException {
             getLog().warn("Report available:");
             getLog().warn(indent(1, patchFile.normalize().toString()));
             getLog().warn("Run 'mvn rewrite:run' to apply the recipes.");
+
+            if (failOnDryRunResults) {
+                throw new MojoExecutionException("Result changes detected. Please see result file for more information.");

```

Comment:
Feel like this sentence could use an edit. "Result changes detected" in particular

Reply:
This changes all of it, but how about:

>Applying recipes would make changes. See logs for more details.

(I can throw this out on slack too for feedback)